static/src/javascripts/projects/common/modules/commercial/creatives/expandable-video-v2.js

```diff
@@ -0,0 +1,100 @@
+define([
+    'bean',
+    'bonzo',
+    'fastdom',
+    'common/utils/_',
+    'common/utils/$',
+    'common/utils/detect',
+    'common/utils/mediator',
+    'common/utils/storage',
+    'common/utils/template',
+    'common/views/svgs',
+    'text!common/views/commercial/creatives/expandable-video-v2.html'
+], function (
+    bean,
+    bonzo,
+    fastdom,
+    _,
+    $,
+	detect,
+    mediator,
+    storage,
+    template,
+    svgs,
+    ExpandableVideoTpl
+) {
+
+    /**
+     * https://www.google.com/dfp/59666047#delivery/CreateCreativeTemplate/creativeTemplateId=10028247
+     */
+    var ExpandableVideo = function ($adSlot, params) {
+        this.$adSlot      = $adSlot;
+        this.params       = params;
+        this.isClosed     = true;
+
+        if (detect.isBreakpoint({min: 'tablet'})) {
+            this.closedHeight = 250;
+            this.openedHeight = 500;
+        } else {
+            this.closedHeight = 150;
+            this.openedHeight = 300;
+        }
+
+    };
+
+    ExpandableVideo.prototype.create = function () {
```

Comment:
This is simply a matter of style, and probably doesn't matter for creative template work, but I'm not sure we should have 'create' methods - 'new' should create a readied object. If a constructor gets too long, it's preferable to decompose it into private functions.